ArduSub/flight_mode.cpp

```diff
@@ -168,8 +168,8 @@ bool Sub::mode_requires_GPS(control_mode_t mode)
     case CIRCLE:
     case POSHOLD:
         return true;
-    default:
-        return false;
+    default: // No valid mode
+        break;
```

Comment:
This one could remain a return false.  The other clauses are returning, sooo....